src/index.js

```diff
@@ -216,7 +216,7 @@ client.on('messageCreate', message => {
       });
     })
   })
-  if (server.isPremium && !(message.member.permissions.serialize().KICK_MEMBERS || message.member.permissions.serialize().BAN_MEMBERS || message.member.roles.cache.some(role => role.id === '332343869163438080'))) {
+  if (!(message.member.permissions.serialize().KICK_MEMBERS || message.member.permissions.serialize().BAN_MEMBERS || message.member.roles.cache.some(role => role.id === '332343869163438080'))) {
```

Comment:
Lets keep this premium for now, as I haven't built ability to set up investigations channel just yet, thus it's being done manually in DB for premium servers right now.

This functionality will be free, but I'll keep toxicity monitoring locked due to performance constraints and API limits.
```suggestion
  if (server.isPremium && !(message.member.permissions.serialize().KICK_MEMBERS || message.member.permissions.serialize().BAN_MEMBERS || message.member.roles.cache.some(role => role.id === '332343869163438080'))) {
```

Reply:
Ah yes of course I didn't mean to push this haha